consul/tests/test_install.sls

```diff
@@ -0,0 +1,18 @@
+{% from "consul/map.jinja" import consul, consul_service with context %}
+
+test_consul_installed:
+  testinfra.file:
+    - name: /usr/bin/consul
+    - exists: True
```

Comment:
It's probably worth verifying the mode of the file as well to make sure that it is executable.